libs/shared-atomic-design-components/src/lib/atoms/FormTextArea.tsx

```diff
@@ -62,43 +86,59 @@ function FormTextArea(props: any) {
           </Form.Help>
         </Columns.Column>
         <Columns.Column>
-          {minChar && (!values[name] || values[name].length < minChar) && (
+          {minChar && !isTouched && !values[name] && charactersLength === 0 && (
             <Content
-              textColor="danger"
+              textColor="grey-dark"
               className="help help--show redi-textarea-characters"
             >
-              {minChar - (values[name] ? values[name].length : 0)} more{' '}
-              {minChar - (values[name] ? values[name].length : 0) > 1
-                ? 'characters'
-                : 'character'}{' '}
-              needed
+              min {minChar} characters required
             </Content>
           )}
+          {minChar &&
+            !isTouched &&
+            values[name] &&
+            charactersLength < minChar &&
+            !minCharAmountReached && (
```

Comment:
Can we encapsulate this conditional into a variable?

From the guide I showed you this morning, there's a tip about precisely this: https://ffg2022redisc-a5j7856.slack.com/archives/C03SJFMDQ14/p1661869990477169

Reply:
Sure. I can't open the link.

Reply:
Ooops, wrong link. Here's the right one: https://github.com/ryanmcdermott/clean-code-javascript#encapsulate-conditionals